db/src/test/query.spec.ts

```diff
@@ -0,0 +1,296 @@
+import test from 'ava';
+import { IllegalArgumentError } from '../errors';
+import { key, all, any, not, value, typedValue, filter, ASC, DESC } from '../query';
+
+function stringifyParse(v: any) {
```

Comment:
```suggestion
function parseStringify(v: any) {
```
(or `stringifyThenParse`, given that some people read it each way).